Source/SwiftLintFramework/Rules/Performance/EmptyArrayRule.swift

```diff
@@ -0,0 +1,34 @@
+import SourceKittenFramework
+
+public struct EmptyArrayRule: ConfigurationProviderRule, OptInRule, AutomaticTestableRule {
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "empty_array",
```

Comment:
should we change this to `empty_collection_literal` and match empty dictionaries too?

Reply:
I like this, should be easy to add.